console/src/persist/state.ts

```diff
@@ -98,6 +98,7 @@ export const open = async <S extends RequiredState>({
   let state = (await db.get<S>(persistedStateKey(version))) ?? undefined;
   if (state != null && migrator != null) {
     try {
+      console.log("applying migrations");
       state = migrator(state);
     } catch (e) {
       console.error("unable to apply migrations. continuing with undefined state.");
```

Comment:
Should this error appear in the console log or as a notification in the Synnax console?

Reply:
Can't make this a notification considering it sets up the code that enables us to display notifications